test/tools/ossfuzz/yulProto.proto

```diff
@@ -398,6 +398,7 @@ message Program {
     BERLIN = 7;
     LONDON = 8;
     PARIS = 9;
+    SHANGHAI = 10;
```

Comment:
Why not add Cancun too?

Reply:
I added cancun here https://github.com/ethereum/solidity/pull/14781

Under the assumption that this would get merged first, and then the other one :-)

Reply:
I mean, we already had Cancun on `develop` so there would have been no problem adding it already in this PR. But since `blobbasefee` is merged now, I guess it does not matter now.